logging/cluster-logging-release-notes.adoc

```diff
@@ -46,42 +46,52 @@ To see these metrics, open the *Administrator* perspective in the {product-title
+
+* * Before this update, the `kibana-proxy` Pod sometimes entered the `CrashLoopBackoff` state and logged the following message `Invalid configuration: cookie_secret must be 16, 24, or 32 bytes to create an AES cipher when pass_access_token == true or cookie_refresh != 0, but is 29 bytes.` The exact actual number of bytes could vary. With this update, generation of the Kibana session secret has been corrected, and the kibana-proxy Pod no longer crash loops due to this error. (link:https://issues.redhat.com/browse/LOG-1446[LOG-1446])
+
+* Before this update, the AWS CloudWatch Fluentd plugin logged its AWS API calls to the Fluentd log at all log levels, thus creating undue pressure on the resources of OCP nodes. With this update, the AWS CloudWatch Fluentd plugin logs AWS API calls only at the "debug" and "trace" log levels. This way, at the default "warn" log level, Fluentd does not consume extra OCP node resources. (link:https://issues.redhat.com/browse/LOG-1071[LOG-1071])
+
+* Before this update, the Elasticsearch OpenDistro security plugin caused user index migrations to fail. This update resolves the issue by providing a newer version of the plugin. Now, index migrations proceed without errors. (link:https://issues.redhat.com/browse/LOG-1276[LOG-1276])
+
+* Before this update, in the *Logging* dashboard in the {product-title} console, the list of top 10 log-producing containers lacked data points. This update resolves the issue, and the dashboard displays the data. (link:https://issues.redhat.com/browse/LOG-1353[LOG-1353])
+
+* Before this update, while the user was tuning the performance of the Fluentd log forwarder by configuring the `chunkLimitSize` and `totalLimitSize` values, the `Setting queued_chunks_limit_size for each buffer to` message reported values that were too low. The current update fixes this issue, so this message reports the correct values. (link:https://issues.redhat.com/browse/LOG-1411[LOG-1411])
+
+* Before this update, the Kibana OpenDistro security plugin caused user index migrations to fail. This update resolves the issue by providing a newer version of the plugin. Now, index migrations proceed without errors. (link:https://issues.redhat.com/browse/LOG-1558[LOG-1558])
+
+* Before this update, using a namespace input filter prevented logs in that namespace from appearing in other inputs. With this update, logs are sent to all inputs that can accept them. (link:https://issues.redhat.com/browse/LOG-1570[LOG-1570])
+
+* Before this update, a missing license file for the viaq/logerr dependency caused license scanners to abort without success. With this update, the viaq/logerr dependency is now licensed under Apache 2.0, resolving the issue. (link:https://issues.redhat.com/browse/LOG-1590[LOG-1590])
```

Comment:
does viaq/logerr need ``?

Reply:
You could think of changing "resolving the issue" to a description, like "enabling license scanners to run successfully".

Reply:
Updated. Thank  you.